src/Microsoft.Framework.PackageManager/Utils/PathUtilities.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Framework.PackageManager
+{
+    internal static class PathUtilities
+    {
+        private static Lazy<string> _runtimeHome = new Lazy<string>(FindRuntimeHome);
+
+        public static string RuntimeHomeFolder
+        {
+            get
+            {
+                return _runtimeHome.Value;
+            }
+        }
+
+        public static string RuntimeBinFolder
+        {
+            get
+            {
+                if (RuntimeHomeFolder == null)
+                {
+                    return null;
+                }
+
+                return Path.Combine(RuntimeHomeFolder, "bin");
+            }
+        }
+
+        private static string FindRuntimeHome()
+        {
+            var runtimeHome = Environment.GetEnvironmentVariable("KRE_HOME");
```

Comment:
Don't use KRE_HOME or KRE_GLOBAL_PATH for install. The install should always be to the user profile `~/.k/bin` irrespective of any of the other settings